docs/atomic.rst

```diff
@@ -28,29 +28,31 @@ In Taichi, augmented assignments (e.g., ``x[i] += 1``) are automatically `atomic
     When atomic operations are applied to local values, the Taichi compiler will try to demote these operations into their non-atomic correspondence.
```

Comment:
'correspondence' has a plural form and personally I prefer 'counterparts' better.

Reply:
well, since this pr is already closed, let me make a new one to fix a few minor issues